sphinx/cmd/quickstart.py

```diff
@@ -245,60 +230,63 @@ def ask_user(d):
 Please enter values for the following settings (just press Enter to
 accept a default value, if one is given in brackets).''')
 
-    if 'path' in d:
+    if d.get('path'):
         print(bold('''
 Selected root path: %s''' % d['path']))
     else:
         print('''
 Enter the root path for documentation.''')
         do_prompt(d, 'path', 'Root path for the documentation', '.', is_path)
 
-    while path.isfile(path.join(d['path'], 'conf.py')) or \
-            path.isfile(path.join(d['path'], 'source', 'conf.py')):
+    while path.isfile(path.join(d.get('path'), 'conf.py')) or \
+            path.isfile(path.join(d.get('path'), 'source', 'conf.py')):
         print()
         print(bold('Error: an existing conf.py has been found in the '
                    'selected root path.'))
         print('sphinx-quickstart will not overwrite existing Sphinx projects.')
         print()
         do_prompt(d, 'path', 'Please enter a new root path (or just Enter '
                   'to exit)', '', is_path)
-        if not d['path']:
+        if not d.get('path'):
             sys.exit(1)
 
-    if 'sep' not in d:
+    if not d.get('sep'):
         print('''
 You have two options for placing the build directory for Sphinx output.
 Either, you use a directory "_build" within the root path, or you separate
 "source" and "build" directories within the root path.''')
         do_prompt(d, 'sep', 'Separate source and build directories (y/n)', 'n',
                   boolean)
 
-    if 'dot' not in d:
+    if not d.get('dot'):
```

Comment:
Is this question asked without `-q` option? I worried this might be skipped if argparse gives a default value.

Reply:
That's a good point. I missed this one.